assets/scss/_custom.scss

```diff
@@ -96,6 +96,25 @@ body.td-home main[role="main"] > section:first-of-type .content p:first-child {
   }
 }
 
+.section-feature#kubeweekly {
+
+  .kubeweekly-signup {
```

Comment:
We can do `> .kubeweekly-inner { h5` to style the `h5` block instead of assigning a CSS class.